packages/fxa-payments-server/src/components/PaymentProviderDetails/index.test.tsx

```diff
@@ -17,7 +17,9 @@ describe('PaymentProviderDetails', () => {
 
     it('Renders credit card logo and last 4 digits', () => {
       const { queryByTestId } = subject();
-      expect(queryByTestId('card-logo-and-last-four')).toBeInTheDocument();
+      const element = queryByTestId('card-logo-and-last-four');
+      expect(element).toBeInTheDocument();
+      expect(element).toHaveClass(`${Customers.CUSTOMER.brand?.toLowerCase()}`);
```

Comment:
I added a single line to this test to assert that `customer.brand` is added as a class, but I’m not sure how else I may want to test this, if at all. This would also have passed before my patch, so I welcome any thoughts.